src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_workload_commands.py

```diff
@@ -0,0 +1,916 @@
+
+        self.cmd('backup job wait -v {vault} -g {rg} -n {job}')
+
+        self.cmd('backup item show -g {rg} -v {vault} -c {container1} -n {item} --backup-management-type AzureWorkload', checks=[
+            self.check('properties.friendlyName', '{fitem}'),
+            self.check('properties.protectedItemHealthStatus', 'Healthy'),
+            self.check('properties.protectionState', 'Protected'),
+            self.check('properties.protectionStatus', 'Healthy'),
+            self.check('resourceGroup', '{rg}')
+        ])
+
+        self.cmd('backup protection disable -v {vault} -g {rg} -i {item} -c {name} --backup-management-type AzureWorkload -y', checks=[
+            self.check("properties.entityFriendlyName", '{fitem}'),
+            self.check("properties.operation", "DisableBackup"),
+            self.check("properties.status", "Completed"),
+            self.check("resourceGroup", '{rg}')
+        ])
+
+        self.cmd('backup item show -g {rg} -v {vault} -c {container1} -n {item} --backup-management-type AzureWorkload', checks=[
+            self.check("properties.friendlyName", '{fitem}'),
+            self.check("properties.protectionState", "ProtectionStopped"),
+            self.check("resourceGroup", '{rg}')
+        ])
+
+    def test_backup_wl_sql_auto_protection(self):
+
+        self.kwargs.update({
+            'vault': vault_sql,
+            'name': container_sql,
+            'fname': container_friendly_sql,
+            'policy': 'HourlyLogBackup',
+            'wt': 'MSSQL',
+            'sub': sub_sql,
+            'default': 'HourlyLogBackup',
+            'rg': rg_sql,
+            'item': item_auto_sql,
+            'fitem': item_auto_sql.split(';')[-1],
+            'id': id_sql,
+            'item_id': item_id_sql,
+            'pit': 'SQLInstance',
+            'entityFriendlyName': backup_entity_friendly_name_sql
+        })
+
+        self.cmd('backup container register -v {vault} -g {rg} --backup-management-type AzureWorkload --workload-type {wt} --resource-id {id}')
+
+        self.cmd('backup container list -v {vault} -g {rg} --backup-management-type AzureWorkload', checks=[
+            self.check("length([?name == '{name}'])", 1)])
+
+        # self.kwargs['protectable_item'] = self.cmd('backup protectable-item show --vault-name {vault} -g {rg} --protectable-item-type {pit} --server-name {fname} --name {item} --workload-type {wt}').get_output_in_json()
+
+        # self.kwargs['protectable_item_name'] = self.kwargs['protectable_item']['name']
+
+        # self.kwargs['protectable_item'] = json.dumps(self.kwargs['protectable_item'], separators=(',', ':')).replace('"', '\\"')
+
+        self.cmd('backup protection auto-enable-for-AzureWL -v {vault} -g {rg} -p {policy} --protectable-item-name {item} --protectable-item-type {pit} --server-name {fname} --workload-type {wt}')
+
+        self.cmd('backup protection auto-disable-for-AzureWL -v {vault} -g {rg} --item-name {item}')
```

Comment:
It should be `auto-disable-for-azurewl`.

Reply:
Ok